source/common.h

```diff
@@ -104,6 +104,15 @@ struct comment
 //
 struct error
 {
+    // Controls whether we print `FILE:line:col: error:` or `FILE(line,col):
+    // error:`.  Need to integrate with editors that jump to errors, etc.
+    // TODO: This should become a compiler flag.
+#ifdef _MSC_VER
+    static constexpr bool print_colon_errors = false;
+#else
+    static constexpr bool print_colon_errors = true;
+#endif
```

Comment:
I don't think we need two defaults... the default should be the same regardless of the compiler we used to build cppfront itself, right?

Reply:
As you say in the comment, we would want to add a switch to cppfront itself, something like this (consistent style with the other flags):

```
static auto flag_print_colon_errors = false;
static cmdline_processor::register_flag cmd_enable_source_info(
    2,
    "format-colon-errors",
    "Emit ':line:col:' format for error messages",
    []{ flag_print_colon_errors = true; }
);
```

Thanks!

Reply:
I updated the code.